frontend/build.sbt

```diff
@@ -113,8 +111,4 @@ libraryDependencies ++= frontendDependencies.map {
 
 addCommandAlias("devrun", "run 9100")
 
-libraryDependencies ++= acceptanceTestDependencies
-
 addCommandAlias("fast-test", "testOnly -- -l Acceptance")
-
-addCommandAlias("acceptance-test", "testOnly acceptance.JoinSupporterSpec")
```

Comment:
where did the actual file go from `main`? https://github.com/guardian/membership-frontend/blob/868fcf6617a1bbb946ea98cb3342f3f44b32ffc4/frontend/test/acceptance/JoinSupporterSpec.scala

Reply:
I think it was deleted quite a while ago. It's certainly not in main now.